modnet/preprocessing.py

```diff
@@ -7,17 +7,19 @@
 
 """
 
+from __future__ import annotations
```

Comment:
Just found out about this, it lets you defer type annotation evaluation so e.g. methods of `MODData` can use `MODData` as a type-hint